GSA/digital-gov-drupal
# DIGITAL-107: Add missing 'External URL' field to short post

> [!NOTE]
> This is replaced by https://github.com/GSA/digital-gov-drupal/pull/49

## Jira ticket
[DIGITAL-107](https://cm-jira.usa.gov/browse/DIGITAL-107)

## Purpose
Adds the `external_url` field to the short post content type, this was missed during the initial setup.
Updated default content short posts with URL.

## Deployment and testing




### QA/Testing instructions

Given I click the short post content from the menu
When I visit the create short page page
Then I should see the external URL field in the form

## Checklist for the Developer




## Checklist for the Peer Reviewers
